backtester/report/report.go

```diff
@@ -68,78 +86,35 @@ func (d *Data) GenerateReport() error {
 	defer func() {
 		err = f.Close()
 		if err != nil {
-			log.Error(log.BackTester, err)
+			log.Error(common.Report, err)
 		}
 	}()
 
 	err = tmpl.Execute(f, d)
 	if err != nil {
 		return err
 	}
-	log.Infof(log.BackTester, "successfully saved report to %v", filepath.Join(d.OutputPath, fileName))
+	log.Infof(common.Report, "successfully saved report to %v", filepath.Join(d.OutputPath, fileName))
 	return nil
 }
 
-// CreateUSDTotalsChart used for creating a chart in the HTML report
-// to show how much the overall assets are worth over time
-func (d *Data) CreateUSDTotalsChart() []TotalsChart {
-	if d.Statistics.FundingStatistics == nil || d.Statistics.FundingStatistics.Report.DisableUSDTracking {
-		return nil
-	}
-
-	usdTotalChartPlot := make([]ChartPlot, len(d.Statistics.FundingStatistics.TotalUSDStatistics.HoldingValues))
-	for i := range d.Statistics.FundingStatistics.TotalUSDStatistics.HoldingValues {
-		usdTotalChartPlot[i] = ChartPlot{
-			Value:     d.Statistics.FundingStatistics.TotalUSDStatistics.HoldingValues[i].Value.InexactFloat64(),
-			UnixMilli: d.Statistics.FundingStatistics.TotalUSDStatistics.HoldingValues[i].Time.UTC().UnixMilli(),
-		}
-	}
-
-	response := make([]TotalsChart, len(d.Statistics.FundingStatistics.Items)+1)
-	response[0] = TotalsChart{
-		Name:       "Total USD value",
-		DataPoints: usdTotalChartPlot,
-	}
-
-	for i := range d.Statistics.FundingStatistics.Items {
-		plots := make([]ChartPlot, len(d.Statistics.FundingStatistics.Items[i].ReportItem.Snapshots))
-		for j := range d.Statistics.FundingStatistics.Items[i].ReportItem.Snapshots {
-			plots[j] = ChartPlot{
-				Value:     d.Statistics.FundingStatistics.Items[i].ReportItem.Snapshots[j].USDValue.InexactFloat64(),
-				UnixMilli: d.Statistics.FundingStatistics.Items[i].ReportItem.Snapshots[j].Time.UTC().UnixMilli(),
-			}
-		}
-		response[i+1] = TotalsChart{
-			Name:       fmt.Sprintf("%v %v %v USD value", d.Statistics.FundingStatistics.Items[i].ReportItem.Exchange, d.Statistics.FundingStatistics.Items[i].ReportItem.Asset, d.Statistics.FundingStatistics.Items[i].ReportItem.Currency),
-			DataPoints: plots,
-		}
-	}
-
-	return response
-}
-
-// CreateHoldingsOverTimeChart used for creating a chart in the HTML report
-// to show how many holdings of each type was held over the time of backtesting
-func (d *Data) CreateHoldingsOverTimeChart() []TotalsChart {
-	if d.Statistics.FundingStatistics == nil {
-		return nil
+func generateFileName(configNickName, strategyName string) (string, error) {
```

Comment:
It would be nice to export this so that the config builder file generation can use it too:

![image](https://user-images.githubusercontent.com/4685270/176585170-5fbdd992-ffe1-4fb9-b865-8de4748691c1.png)